cmd/dummy/main.go

```diff
@@ -22,9 +23,12 @@ type ServeCommand struct {
 	DataServer       string `long:"data-server" default:"ipv4://localhost:10301" env:"LOOKOUT_DATA_SERVER" description:"gRPC URL of the data server"`
 	RequestUAST      bool   `long:"uast" env:"LOOKOUT_REQUEST_UAST" description:"analyzer will request UAST from the data server"`
 	RequestFilesPush bool   `long:"files" env:"LOOKOUT_REQUEST_FILES" description:"on push events the analyzer will request files from HEAD, and return comments"`
+	ProbesAddr       string `long:"probes-addr" default:":8091" env:"LOOKOUT_ANALYZER_PROBES" description:"TCP address to bind the health probe endpoints"`
```

Comment:
This env name should have a more descriptive name. If it expects and address, then I would use something like `LOOKOUT_ANALYZER_PROBES_ADDRESS`

Another thing I don't understand well is why default is in a different format as `LOOKOUT_DATA_SERVER`

Reply:
The different format is because `DATA_SERVER` is for a grpc url, this one is the address to bind the http server.

I'll change it to `LOOKOUT_ANALYZER_PROBES_ADDRESS` and `0.0.0.0:8091` to make it more clear.

Reply:
Done in 80c65d3.